src/Styleguide/Pages/Artist/ArtistApp.tsx

```diff
@@ -16,22 +15,15 @@ import { LoadingArea } from "./Components/LoadingArea"
 // Inner content max width 1112
 
 export interface ArtistAppProps {
-  artist: ArtistHeader_artist
+  artist: any
```

Comment:
I dropped this b/c we also query for tabs here, it might be cleaner to move the tab nav into the `ArtistHeader` component?

Reply:
I think we should leave it here since it contains other content and helps make the overall page structure clear.